app/controllers/admin/admin_invitations_controller.rb

```diff
@@ -48,6 +48,9 @@ def find
     elsif !invitation_params[:invitee_email].blank?
       @invitations = Invitation.where('invitee_email LIKE ?', "%#{invitation_params[:invitee_email]}%")
       @invitation = @invitations.first if @invitations.length == 1
+      if @invitations.length == 0
+	      @invitations = nil
```

Comment:
Hound is right -- [we use spaces, not tabs](https://github.com/otwcode/otwarchive/wiki/Coding-standards#general-code-formatting), so this should be:
```ruby
      if @invitations.length == 0
        @invitations = nil
      end
```
But it might also be possible to keep it on one line, like on line 50:
```ruby
      @invitations == nil if @invitations.length == 0
```
But it looks to me like it _might_ be possible to resolve this issue with even less code, by changing `@invitations` on old line 52/new line 55 to `@invitations.present?`. Is that something you tried and ran into issues with?